.env.example

```diff
@@ -25,15 +25,36 @@ export template_dir="."             # Directory path for templates
 
 # Related to WhatsApp Business and Meta (leave empty if you're not planning to use WhatsApp)
 # Source 1: https://www.youtube.com/watch?v=KP6_BUw3i0U
+#   Watch Until 32:25
 # Source 2: https://glitch.com/edit/#!/insidious-tartan-alvarezsaurus
 # Source 3: https://developers.facebook.com/blog/post/2022/10/24/sending-messages-with-whatsapp-in-your-python-applications/#u_0_39_8q
+
+# Moreover, if want to test whatsapp's webhook locally, you can use zrok on a reserved URL with a zrok "share token" 
+# obtained by contacting its current holder: https://github.com/OdyAsh (source 1, 2 below)
+# Alternatively, you can change the webhook url all together (source 3, 4 below)
+# Check these sources for more details: 
+# Source 1: https://dev.to/odyash/quickly-share-your-app-with-zrok-4ihp
+# Source 2: https://openziti.discourse.group/t/how-do-i-use-a-reserved-share-on-different-devices/2379/2
+# Source 3: https://youtu.be/KP6_BUw3i0U?t=1294 
+#   (@21:33 and 25:30, however they use glitch instead of zrok, so you'll just need to change the webhook url to your zrok url)
+# Source 4 (where you can change callback url, given that your facebook account gets access by the app's admins): 
+#   https://developers.facebook.com/apps/871020755148175/whatsapp-business/wa-settings/
+#   Note 1: Obviously, that `871...175` is the testing app's public id, so if this link still doesn't work even after you gain access, 
+#   then the admins most probably created a new test app instance
+#   Note 2: If an unexpected 3rd party discovers the ZROK_SHARE_TOKEN, 
+#   a new one will have to be generated, then added to Meta's callback URL of the *testing* app
+#   (Noting that the *production* app's callback URL will be different anyway, so the 3rd party won't be able to access that app)
+#   (but we still don't want random calls to be made to our testing app, so that's why we'll still have to change an exposed token :])
+
```

Comment:
Love the extensive comments.

Reply:
Thank you! I was thinking of other ways to document these comments externally, but until we figure something out, will stick to inline commenting as I do think these should help newcomers .